src/test/scala/Wishbone/WishboneTest.scala

```diff
@@ -0,0 +1,39 @@
+package flexpret.Wishbone
+
+import chisel3._
+import chiseltest._
+import flexpret.core.{FlexpretConfiguration, InstMemConfiguration}
+import org.scalatest._
+
+class WishboneTest extends FlatSpec with ChiselScalatestTester {
+
+  behavior of "Wishbone"
+
+  def write(c: WishboneMaster, addr: Int, data: Int): Unit = {
+    c.busIO.data_in.poke(data.U)
+    c.busIO.write.poke(true.B)
+    c.busIO.enable.poke(true.B)
+    c.busIO.addr.poke(addr.U)
+  }
+
+  def read(c: WishboneMaster, idx: Int, addr: Int, expect: Int) = {
```

Comment:
I'm confused. Is there meant to be a function body here?

Reply:
Yes at some point

Reply:
Please document?